blog/releases/announcing-knative-v0-22-release.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Version v0.22 release"
+linkTitle: "Version v0.22 release"
+Author: "[Carlos Santana](https://twitter.com/csantanapr)"
+Author handle: https://github.com/csantanapr
+date: 2021-04-06
+description: "Knative v0.22 release announcement"
+type: "blog"
+image: knative-eventing.png
+---
+
+
+### Announcing Knative v0.22 Release
+
+A new version of Knative is now available across multiple components.
+Follow the instructions in the documentation [Installing Knative](https://knative.dev/docs/install/) for the respective component.
+
+#### Table of Contents
+- [Serving v0.22](#serving-v022)
+- [Eventing v0.22](#eventing-v022)
+- Eventing Extensions
+    - [Eventing Kafka Broker v0.22](#eventing-kafka-broker-v022)
+- `kn` [CLI v0.22](#client-v022)
+    - `kn` [CLI Plugins](#cli-plugins)
+- [Operator v0.22](#operator-v022)
+- [Thank you contributors](#thank-you-contributors)
+
+
+
+### Highlights
+
+- Serving Domain Mapping improves multi-tenency support to avoid domain name 'squatting' in cluster.
+- Eventing now allows Trigger users to set another namespace's Subscriber
+- Eventing kafka broker now Kubernetes's minimum version is 1.18
+- Eventing kafka now supports the ability to choose between ordered and unordered delivery
+- The CLI `kn` 0.22.0 comes with some bug fixes and minor feature enhancements. It's mostly a polishing release. If you are using the client API, there is a breaking change that was needed to align with Kubernetes client API's
+- There are two new CLI plugins align with 0.22 release, [kn-plugin-admin](https://github.com/knative-sandbox/kn-plugin-admin) and [kn-plugin-source-kafka](https://github.com/knative-sandbox/kn-plugin-source-kafka)
+- The Knative Operator release with some bug fixes, and supports the latest version of knative serving and eventing.
+
+
+### Serving v0.22
+
+#### 💫 New Features & Changes
+
+- Added an autoscaling annotation to choose a different aggregation algorithm for the autoscaling metrics. This is experimental currently. [#10840](https://github.com/knative/serving/pull/10840)
+- Added autocreateClusterDomainClaims flag to network config map. [networking/#330](https://github.com/knative/networking/pull/330)
+
+#### 🐞 Bug Fixes
+
+- Adds validation that a default max-scale is set if a max-scale-limit is specified in the autoscaler configmap (since otherwise the default max-scale, i.e. 0 = no max, would fail validation as it is above the max-scale-limit). [#10921](https://github.com/knative/serving/pull/10921)
```

Comment:
```suggestion
- Added validation that a default `max-scale` is set if a `max-scale-limit` is specified in the autoscaler ConfigMap. Otherwise, the default `max-scale`, `0` = no max, would fail validation as it is above the `max-scale-limit`. [#10921](https://github.com/knative/serving/pull/10921)
```